app/lib/paymentUtils.js

```diff
@@ -30,8 +31,45 @@ function hasTemporaryPremiumAccess() {
   return storage.load(`${TEMPORARY_PREMIUM_KEY}-${me.get('_id')}`)
 }
 
+function setTrackedPremiumPurchase() {
+  storage.save(`${TEMPORARY_PREMIUM_KEY}-${me.get('_id')}`, true, 6 * 60)
+}
+
+function hasTrackedPremiumAccess() {
+  return storage.load(`${TRACKED_PREMIUM}-${me.get('_id')}`)
+}
+
+function getTrackingData ({ amount, duration }) {
+  const options = {}
+  if (amount) {
+    const numericalAmount = parseInt(amount) / 100
+    options.value = `${numericalAmount}`
+    options.currency = 'USD'
+    // ltv -> calculate based on duration
+    options.predictedLtv = `${numericalAmount} * ${getLtvMultiplier(duration)}`
```

Comment:
Facebook will want this called `predicted_ltv`, I think. Note that the Facebook tracker needs to add event properties for standard events, in general.

For GA, and possibly others, we actually want `value` to be the predicted LTV.

So maybe we can, in the event options here, name these something unambiguous, like `purchaseAmount` and `predictedLtv`, and then in the GA tracker, we can say, `properties.value = properties.predictedLtv`, and in the FB tracker, we can say, `properties.value = properties.amount; properties.predicted_ltv = properties.predictedLtv` or something like this?